### PR TITLE
Feature/template meta tags

### LINKED ISSUE #675: Template more meta tags
Meta tags may be overridden in the Django templates by using `{% block extrametatags %}`.

Add overrides for destination details and add more tag overrides to articles.

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Customize social sharing meta tags for article and destination details views. Closes #675. Will need to deploy to staging to test (can use [Twitter card validator](https://cards-dev.twitter.com/validator)).

### CODE REVIEW

File: python/cac_tripplanner/templates/learn-detail.html [hunk 1]
```diff
@@ -4,8 +4,24 @@
 {% block pagetitle %}
 <title>{% block title %}GoPhillyGo | {{ article.title }}{% endblock %}</title>
 
-<meta property="og:title" content="GoPhillyGo | {{ article.title }}" />
-<meta name="twitter:title" content="GoPhillyGo | {{ article.title }}" />
+<meta property="og:title" content="GoPhillyGo | {{ article.title|striptags }}" />
+<meta name="twitter:title" content="GoPhillyGo | {{ article.title|striptags }}" />
+{% endblock %}
+
+{% block extrametatags %}
```
Comment (CloudNiner) [inline python/cac_tripplanner/templates/learn-detail.html]: It took a surprisingly long time for my brain to parse what this says.
:medal_sports: for me.

File: python/cac_tripplanner/templates/place-detail.html [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+<title>{% block title %}GoPhillyGo | {{ destination.name }}{% endblock %}</title>
+
+<meta property="og:title" content="{{ destination.name }}" />
+<meta name="twitter:title" content="{{ destination.name }}" />
+{% endblock %}
+
+{% block extrametatags %}
+<meta property="fb:app_id" content="{{ fb_app_id }}" />
+<meta property="og:url"
+    content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}" />
```
Comment (CloudNiner) [inline python/cac_tripplanner/templates/place-detail.html]: Is there no `request.protocol`? Might be cleaner, but then you have the potential for `file://` to sneak in there. :man_shrugging:

Comment (flibbertigibbet) [inline python/cac_tripplanner/templates/place-detail.html]: Not directly, as far as I am aware. `is_secure` seems to be [the recommended](http://stackoverflow.com/questions/1841826/can-you-check-the-internet-protocol-from-djangos-template) thing to use.

File: python/cac_tripplanner/templates/learn-detail.html (hunk 1 above)
Comment (flibbertigibbet) [inline python/cac_tripplanner/templates/learn-detail.html]: I'll be sure not to add an `extremetags` section :black_joker:

**Comment by flibbertigibbet** [comment]
Thanks for the reviews! Meant to only assign one of you, but couldn't find a way to un-assign for review.

**Pull request merged**